server/coverage/index.html

```diff
@@ -0,0 +1,162 @@
+<!doctype html>
```

Comment:
Please don't push coverage files. I don't think they need to be in your Repository